src/styles/blocks/content.scss

```diff
@@ -0,0 +1,10 @@
+.content {
+  @extend %flex-center;
+
+  height: 100vh;
+
+  background-image: url(../images/dynamite.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
```

Comment:
you can use them together in one line :)

Reply:
Oh, thanks! I'll change it